tests/testCameraMapper.py

```diff
@@ -390,15 +392,39 @@ def testPackageName(self):
 
 class ParentRegistryTestCase(unittest.TestCase):
 
+    @staticmethod
+    def _createRegistry(path):
+        cmd = """CREATE TABLE x(
+           id INT,
+           visit INT,
+           filter TEXT,
+           snap INT,
+           raft TEXT,
+           sensor TEXT,
+           channel TEXT,
+           taiObs TEXT,
+           expTime REAL
+           );
+        """
+        conn = sqlite3.connect(path)
+        conn.cursor().execute(cmd)
+        conn.commit()
+        conn.close()
+
     def setUp(self):
         self.ROOT = tempfile.mkdtemp(dir=ROOT, prefix="ParentRegistryTestCase-")
         self.repoARoot = os.path.join(self.ROOT, 'a')
         args = dafPersist.RepositoryArgs(root=self.repoARoot, mapper=MinMapper1)
         butler = dafPersist.Butler(outputs=args)
-        with open(os.path.join(self.repoARoot, 'registry.sqlite3'), 'w') as f:
-            f.write('123')
+        self._createRegistry(os.path.join(self.repoARoot, 'registry.sqlite3'))
+        del butler
```

Comment:
This might be needed, but keeping it means that the linter won't complain about `butler` only being used once.